File: Pocs/Automated Test/AgileContent/Node.js/SeleniumMocha/Personal/test/personal-test.js
const driverFactory = require('../base/driver-factory');
const config = require('../config/config');
const HomePage = require('../pages/home-page');
const LoginPage = require('../pages/login-page');
const MisContenidosPage = require('../pages/mis-contenidos-page');
const assert = require('assert');

describe('Personal-QA', function() {
  this.timeout('300s');

  const driver = driverFactory.build(); 
 
  before(async function(){ 
    await driver.get(config.baseURL);
    await driver.sleep(2000); // ¯\_(ツ)_/¯
  });

  after(async function(){
    await driver.quit();
  });

  it('should login, go to Mis Contenidos and check biblioteca block is visible.', async function() {
    
    const homePage = new HomePage(driver);
    await homePage.ingresar();

    const loginPage = new LoginPage(driver);
    await loginPage.login(config.credentials);

    await homePage.goToMiBiblioteca();

    const misContenidosPage = new MisContenidosPage(driver);
    assert.equal(await misContenidosPage.geTtitleContainerText(), 'Mi biblioteca');    
  }); 
});

File: Pocs/Automated Test/AgileContent/NetCoreStandard/NetCoreSeleniumTest/Test/PersonalTest.cs
using NUnit.Framework;
using Selenium.Utils;
using NetCoreSeleniumTest.Pages;

namespace NetCoreSeleniumTest.Test
{
    public class PersonalTest:BaseTest
    {
        [SetUp]
        public void Setup()
        {
            _driver.GoToUrl(baseUrl);
        }

        [Test]
        public void Test()
        {
            //Should login, go to Mis Contenidos and check biblioteca block is visible.

            var homePage = new HomePage(_driver);

            homePage.Ingresar();

            var loginPage = new LoginPage(_driver);
            loginPage.Login(currentUser);

            homePage.GoToBiblioteca();

            var misContenidosPage = new MisContenidosPage(_driver);
            Assert.AreEqual("Mi biblioteca", misContenidosPage.GeTtitleContainerText());
        }

        [TearDown]
        public void TearDown()
        {
            _driver.Quit();
        }
    }
}
